src/main/resources/application.yaml

```diff
@@ -84,10 +84,11 @@ quarkus:
           policy: authenticated
 
     limits:
-      # Let's limit low-level maximum HTTP request size to 5 megs: stricter limit (1 meg)
-      # is applied at the JSON API level. Low-level limits may result in EPIPE/413 errors
+      # Let's limit low-level maximum HTTP request size to 20 megs: stricter
+      # limit (4 meg per document) is applied at the JSON API level.
+      # Low-level limits may result in EPIPE/413 errors
       # whereas at higher level we can use regular JSON API error responses
-      max-body-size: 5M
+      max-body-size: 20M
```

Comment:
Document sizes are still checked by shredder separately, and 4 meg limit is for document regardless of indexing. This limit is at lower, HTTP level, mostly to prevent someone from sending say 1 gig request. If so, we'd read it all in memory, do processing, and only later on hit per-document limit.

So this is just for DoS reasons. Specific value to use is somewhat arbitrary; could use larger value if we want to (f.ex to allow larger payloads for insertMany)